.circleci/config.yml

```diff
@@ -48,6 +48,8 @@ jobs:
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               login="$(aws ecr get-login)"
               ${login}
-              docker tag app "${ECR_ENDPOINT}/app:${CIRCLE_SHA1}"
-              docker push "${ECR_ENDPOINT}/app:${CIRCLE_SHA1}"
+              # create="${aws ecr create repository --repository-name ${CIRCLE_PROJECT_REPONAME}}"
+              # ${create}
```

Comment:
Yes, that was a question i was going to bring up, when does the repo get created? The creds cicleci uses at the moment only has rights to push and not create. If it will get created at an earlier point yes happy to take out the those lines as they will not be needed, for now the ecr end point of an env var